docs/ref/models/instances.txt

```diff
@@ -871,3 +885,8 @@ Other attributes
     since they are yet to be saved. Instances fetched from a ``QuerySet``
     will have ``adding=False`` and ``db`` set to the alias of the associated
     database.
+
+    .. note::
+
+        Whilst an underscore prefix is typically used to indicate a private
+        attribute, the documented parts of ``_state`` are a public, stable API.
```

Comment:
I don't see much value in such notes, `_state` is already documented so I don't think we need to encourage users or assure about stability. It's clearly stated in [API stability docs](https://docs.djangoproject.com/en/3.0/misc/api-stability/#what-stable-means).

Reply:
Yeah I guess that's right. I'm happy to remove them.